temba/channels/utils.py

```diff
@@ -0,0 +1,220 @@
+from __future__ import absolute_import, unicode_literals
+from django.http.response import HttpResponse
+from twilio.util import RequestValidator
+from temba import settings
+from .models import Channel
+from temba.contacts.models import URN, Contact
+from temba.flows.models import FlowRun, Flow
+from temba.ivr.models import IVRCall
+from temba.msgs.models import Msg
+from temba.triggers.models import Trigger
+from twilio import twiml
+from django.core.exceptions import ValidationError
+
+__author__ = 'teehamaral'
+
+
+class TwilioPostHandler(object):
```

Comment:
Not quite sure I understand this approach.. this handler isn't being used anywhere else is it?

The right approach here is to have a single base class for say TwimlAPIHandler and then have TwilioHandler extend it and override a few of the methods. There's no reason to create this weird object that isn't anything at all, much less in utils.py.